mico-core/src/main/java/io/github/ust/mico/core/resource/OpenFaasResource.java

```diff
@@ -46,7 +46,7 @@
 @RequestMapping(value = OpenFaasResource.OPEN_FAAS_BASE_PATH, produces = MediaTypes.HAL_JSON_VALUE)
 public class OpenFaasResource {
 
-    public static final String OPEN_FAAS_BASE_PATH = "/openFaaS";
+    public static final String OPEN_FAAS_BASE_PATH = "/openfaas";
```

Comment:
This needs to be changed in the api code of the Ui too.

Reply:
oh my fault :(
Will you change it in the UI code?

Reply:
No.

Reply:
I reversed it.
@wagnerdk What was your initial reason to rename it?

Reply:
I think it was to match the UI code.

Reply:
If anything the UI code should follow the mico core code and not the other way around.

Reply:
I would like to change it to `/openfaas/externalAddress` (both the UI and the backend). Ok?